packages/dapp/src/components/Listings.tsx

```diff
@@ -39,19 +40,29 @@ class Listings extends React.Component<{}, ListingsState> {
     return (
       <StyledDiv>
         whaddup listings: {this.state.applications.toString()}
+        <br/>
+        {this.state.error && this.state.error}
```

Comment:
this is a low-fi way to say "if this.state.error is defined, render this.state.error". When I build the actual component, this will be done in a nicer way.

https://reactjs.org/docs/conditional-rendering.html

Reply:
you don't need the double for that if it's undefined or null it won't render anything if its defined it will render itself.

Reply:
that makes sense, fixed